pkg/apis/core/types.go

```diff
@@ -2693,6 +2693,12 @@ type PodSpec struct {
 	// If not specified, the pod will not have a domainname at all.
 	// +optional
 	Subdomain string
+	// If set to true, the fully qualified domain name (FQDN) of a Pod is set as hostname of its containers.
```

Comment:
Reword for clarity, maybe :

If true the pod's hostname will be configured as the pod's FQDN, rather than the leaf name (the default).

Reply:
changing, thanks